scss/inc/_navigatorFizzy.scss

```diff
@@ -0,0 +1,312 @@
+@import 'inc/bootstrap';
+
+$lineHeight: 3rem;
+$borderWidth: 1px;
+$backgroundColor: #f2f2f2;
+$radiusCircular: 50%;
+$sizeDefault: 4rem;
+$hitboxSize: 6.25rem;
+$borderThin: 0.125rem;
+$colorTextDisabled: hsl(0, 0%, 45%);
+$colorBgDefault: white;
+$borderMedium: 0.25rem;
+$borderMediumPlus: 0.375rem;
+$colorTextDefault: hsl(0, 0%, 12%);
+$colorBgInverted: $colorTextDefault;
+$colorTextInverted: white;
+$colorPrimaryHighlight: hsl(208, 100%, 32%);
+$colorSuccess: hsl(164, 54%, 25%);
+$colorWarning: hsl(356, 65%, 40%);
+$offset: calc((#{$hitboxSize} - #{$sizeDefault}) / 2);
+
+.qti-navigator.fizzy {
+    .qti-navigator-tree.key-navigation-group.focusin {
+        background: none;
+    }
+}
+.qti-navigator.fizzy {
+    @include disableSelect();
+    @include flex-container();
+    @include vendor-prefix(flex-direction, column, property);
+    padding: 0;
+    cursor: default;
+    height: 100%;
+    position: relative;
+    background-color: $backgroundColor !important;
+    width: 29rem;
+
+    /* reset parent style */
+    button [class*=' icon-'] {
+        padding: 0;
+    }
+
+    /* reset parent styles */
+    .qti-navigator-filters.key-navigation-group.focusin.focusin {
+        background: $backgroundColor;
+    }
+
+    /* header */
+    & > .qti-navigator-label {
+        display: flex;
+        justify-content: space-between;
+        margin: 12px 10px 0 10px;
+        padding-bottom: 10px;
+        border-bottom: 1px solid #ccc;
+
+        .qti-navigator-text {
+            @include font-size(20);
+            font-weight: bold;
+        }
+
+        .icon-close {
+            @include font-size(28);
+            line-height: $lineHeight;
+            color: $colorTextDefault;
+
+            &:hover {
+                color: $colorTextDefault;
+                text-decoration: none;
+                cursor: pointer;
+            }
+        }
+    }
+
+    /* tree */
+    .qti-navigator-tree {
```

Comment:
This tree element gets a scrollbar when its content grows (I think when the button on the final line gets an indicator under it). Since the `<aside>` is also scrollable, this caused - in my case - a double scrollbar and a wrapping of the buttons (4 per row became 3 per row). I'm not exactly sure why the `position:absolute` indicator element does this. Maybe we can avoid the situation by leaving extra space for it?
![Screenshot 2021-11-05 at 10 01 03](https://user-images.githubusercontent.com/43652944/140486341-f5b5f93a-c2cd-4376-ab5c-0c44d8586e86.png)

Reply:
I fixed double scroll, but i a little another way. I didn't catch exactly this issue, but you can check.

Reply:
Fix works.